### PR TITLE
add shell commands to simplify life, fix some typos 

### PR DESCRIPTION
**Pull request by apetrychak**
Easier to just run the commands when creating or updating .env files etc. instead of thinking (because of possible time pressure during day 2).

Also fixed some typos.

### CODE REVIEW

File: ex-11/doc/episodes_code_config.md [hunk 1]
```diff
@@ -34,7 +34,13 @@ Steps:
 ---
 ## The Config
 
-* The episodes env file (appsec-course-api-episodes-eq.env) needs to be updated with new config
+* Open appsec-course-api-episodes-eq.env
```
Comment (larskaare) [inline ex-11/doc/episodes_code_config.md:37]: I suggest that we keep the initial line with "The episodes env file ......." and then continue with your suggestions.

**Review by larskaare — APPROVED** [review]
Thanx - great suggestions and good fixing those typos :) I added one comment on keeping an "existing line" ..... besides this we're good to merge from my view

**Review by steinsiv — APPROVED** [review]
Good catches! Looks good for me to merge after the suggestions above have been considered 👍

File: ex-11/doc/episodes_code_config.md (hunk 1 above)
Comment (apetrychak) [reply ex-11/doc/episodes_code_config.md:38]: Hi @larskaare , thanks for feedback :). I've added original line in latest commit.

**Pull request merged**